SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1101,8 +1103,22 @@ void USpatialReceiver::OnComponentUpdate(const Worker_ComponentUpdateOp& Op)
 	case SpatialConstants::NETMULTICAST_RPCS_COMPONENT_ID:
 		HandleRPC(Op);
 		return;
+	case SpatialConstants::TOMBSTONE_COMPONENT_ID:
+		Tombstone* TombstoneComponent = StaticComponentView->GetComponentData<Tombstone>(Op.entity_id);
+		if (TombstoneComponent->bIsDead)
+		{
+			RemoveActor(Op.entity_id);
+			return;
+		}
 	}
 
+	// If this entity has a Tombstone component marked dead, abort all component processing
```

Comment:
I don't think we can guarantee that no other component updates will come in for a particular entity after the Tombstone update, so we may need to guard against it (ex. if some other component is updated in the same oplist).

Thoughts?